src/pages/challenges.tsx

```diff
@@ -1,23 +1,21 @@
-import { FunctionComponent } from 'react';
-import Layout from 'components/Layout';
-import Section from 'components/section';
-import Navbar from 'components/navbar';
-import ChallengesJson from 'mocks/challenges.json';
-import { CHALLENGES_SCREEN } from '../components/constants/display-sections.js';
+import { FC } from 'react';
 import { Helmet } from 'react-helmet';
+import Section from '../components/section';
+import Layout from '../components/Layout';
+import Navbar from '../components/navbar';
+import ChallengesJson from '../mocks/challenges.json';
+import { CHALLENGES_SCREEN } from '../components/constants/display-sections.js';
 
-const Challenges: FunctionComponent = () => {
-  return (
-    <Layout>
-      <Helmet>
-        <title>Challenges | Status Real Dev Squad</title>
-      </Helmet>
-      <Navbar page='DS' />
-      <div>
-        <Section content={ChallengesJson} screen={CHALLENGES_SCREEN} />
-      </div>
-    </Layout>
-  );
-};
+const Challenges: FC = () => (
+  <Layout>
+    <Helmet>
+      <title>Challenges | Status Real Dev Squad</title>
```

Comment:
Doesn't `Status Real Dev Squad` sound a bit weird?
Maybe `Challenges | Status :: Real Dev Squad` ?

Reply:
This is inspired by members site, where we have `Members Read Dev Squad`.
We'll discuss if we really need this change.

Reply:
I think just `Challenges | Real dev squad` will make much more sense.